data-prepper-logstash-configuration/src/test/resources/org/opensearch/dataprepper/logstash/log-ingest-to-opensearch.conf

```diff
@@ -12,7 +12,14 @@ filter {
     }
     drop { }
     kv {
-        target: "test"
+        target => "test"
+    }
+    mutate {
+        add_field => { "message" => 3 "message2" => 4.2 "message3" => "test3" "message4" => true }
```

Comment:
Can you update these tests to not repeat any values? The `message` is used a few times. It would be better to have them all unique to provide certainty that they are all working.